lib/trackingParams.js

```diff
@@ -0,0 +1,40 @@
+/** @module trackingParams */
+const {
+    generateDNRRule
+} = require('./utils')
+
+const TRACKING_PARAM_PRIORITY = 20000
+
+function generateTrackingParameterRules (config) {
+    if (config.features?.trackingParameters?.state !== 'enabled') {
+        return []
+    }
+
+    const allowedDomains = config.features.trackingParameters.exceptions?.map(e => e.domain)
+
+    // Skip any wildcard or regex parameters, we can't support these in MV3
+    const trackingParams = config.features.trackingParameters.settings?.parameters?.filter(param => !(param.match(/[*[\]()]/, 'g')))
```

Comment:
Your regex is just as valid, but perhaps go with `/[*+?{}[\]]/` since that matches what we do for MV2. [See related code](https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/shared/js/background/url-parameters.es6.js#L6-L10).